packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -55,6 +55,10 @@ export interface WizardProps extends React.HTMLProps<HTMLDivElement> {
   onSave?: (event: React.MouseEvent<HTMLButtonElement>) => void | Promise<void>;
   /** Callback function to close the wizard */
   onClose?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  /** @beta Flag indicating whether the wizard content should be focused after the onNext or onBack callbacks
+   * are called.
+   */
+  shouldFocusContentOnNextOrBack?: boolean;
```

Comment:
A little nit picky, but I find this name a little confusing. Is there something that we could use that would be a little clearer?

Reply:
I'd definitely be in favor of updating this prop name, I just couldn't really think of a better alternative at the time 😆 `shouldFocusContentOnStepChange` wouldn't be totally accurate since technically clicking a nav item changes the step, but we don't move focus on that. `shouldFocusContent` would be shorter, but probably less clear? `shouldFocusContentOnFooterAction`?

If it matters the prop is beta, so we should be able to tweak the naming if needed.

Reply:
@tlabaj do you have any suggestions for a potential prop name here?

Reply:
I would vote for the shorter name.  I think `shouldFocusContent` will work as long as description is clear.